src/client/components/HomepageHeader/Header.component.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import robot from '../../assets/images/robot.png';
-import './header.css';
+import './header.styles.css';
```

Comment:
Hi Mohammed.. You did a really good job. Could you please change the above css file name to Header.styles.css

Reply:
Yes, good eye :)